src/Panel/Panel.js

```diff
@@ -148,6 +149,7 @@ export default class Panel extends React.Component {
       header ?
         <PanelHeader
           title={header.hideTitleWhenExpanded && isExpanded ? undefined : header.title}
+          size={header.size || HeaderSize('small')}
```

Comment:
Not really sure about this.
It should be moved inside `defaultProps` of `PanelHeader`, and it won't affect the README.